.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.5
```

Comment:
Yea I bumped to 2.7 because of [Rubocop](https://rubygems.org/gems/rubocop/versions/1.61.0)
![image](https://github.com/Shopify/buildkit/assets/20158582/d3aa8551-e511-4549-a4f4-98e91008024a)

Reply:
Kept the Ruby version at 2.5.5 👍🏻